docs/explanations/how_to_interpret_the_security_status_command.md

```diff
@@ -44,7 +223,7 @@ livepatch:
 Let's understand what each key means in the output of the `pro security-status`
```

Comment:
```suggestion
Let's understand what each key means in the output of the `pro security-status --format yaml`
```

Reply:
Since we earlier discussed the plain `pro security-status` command, I think we need to add this qualifier for the sake of clarity